services/datamanager/builtin/sync/upload_data_capture_file.go

```diff
@@ -211,17 +372,32 @@ func sendStreamingDCRequests(
 	return nil
 }
 
-func getFileExtFromImageFormat(res pb.Format) string {
-	switch res {
-	case pb.Format_FORMAT_JPEG:
+func getFileExtFromImageFormat(t cameraPB.Format) string {
+	switch t {
+	case cameraPB.Format_FORMAT_JPEG:
 		return ".jpeg"
-	case pb.Format_FORMAT_PNG:
+	case cameraPB.Format_FORMAT_PNG:
 		return ".png"
-	case pb.Format_FORMAT_RAW_DEPTH:
+	case cameraPB.Format_FORMAT_RAW_DEPTH:
```

Comment:
I don't think so... `cameraPB` is an alias to `go.viam.com/api/component/camera/v1` which is what `pb` was also an alias to in the old version... I just changed it b/c I needed to be able to refer to `datasyncPB` as well

Reply:
That sounds good. Saw the ticket come in and the change for 1 elsewhere in this PR. My only concern is whether it's a good idea to add information as an image tag/annotation that the user did not explicitly ask us to include. Seems like a new pattern - passing information to the user through a tag - that might be unexpected for users. I feel like additional parameters would be more appropriate although we don't have a way to dynamically add these params at capture time. Maybe this is fine, but wanted to call it out.

Reply:
I don't think there is much difference between the source name of each camera and a classification from an ML model. In either case it is a string that contains metadata about the image. IMO for any resource method response that contains both tabular and binary data we will need to have some place to put the tabular data when we sync that response to the datasync api. I think the user IS asking to be able to distinguish which camera sensor each image in the response came from when they query their data in binary data by filter.

Also, for the intel realsense (the main reason why this method exists) the `souce_name` strings are configured by the user: https://github.com/viam-modules/viam-camera-realsense in the robot config:
 the `sensors` list becomes the source_names in the GetImages response.

Reply:
Yeah that's fair. I guess CaptureAllFromCamera feels different to me because a classification is typically an annotation on an image, whereas us returning this metadata as a tag might be unexpected. But given this is the only avenue we have at the moment for returning arbitrary metadata along with the image, I'm okay with this for now.